frontend/src/pages/article/components/author/styles.module.scss

```diff
@@ -0,0 +1,90 @@
+.container {
+  max-width: 466px;
+  background: #ebece7;
```

Comment:
Use a variable for color here

Reply:
This is author-component from task/wr-53. I have merge those branch when worked on my task. I will merge those branch again with changes after review. I already linked your comments to wr-53.